src/uberjob/_execution/run_function_on_graph.py

```diff
@@ -133,13 +135,10 @@ def process_node(node):
                     remaining_predecessor_count.value -= 1
                     should_submit = remaining_predecessor_count.value == 0
                 if should_submit:
-                    submit(successor)
+                    queue.put(successor)
 
-    queue = create_queue(graph, scheduler)
-    with worker_pool(queue, process_node, worker_count) as submit:
+    with worker_pool(queue, process_node, worker_count):
         try:
-            for node in source_nodes(graph):
```

Comment:
The create_queue function now takes initial_items as an argument, and there was never a need to introduce a variable called "submit" which is just an alias for queue.put.